rele/client.py

```diff
@@ -49,7 +49,7 @@ def __init__(self):
             credentials=settings.RELE_GC_CREDENTIALS)
 
     def publish(self, topic, data, **attrs):
-        data = JSONRenderer().render(data)
+        data = json.dumps(data).encode()
```

Comment:
Im afraid this wont work in the case that you pass in `serializers.data`. We tried this, and thats why we had to use `data = JSONRenderer().render(data)`. If anything, you will would have to pass in your data already encoded to the publish function, or we make the encoding method configurable.

Reply:
Did you try passing `serializer.to_representation()` ?